sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -79,37 +79,53 @@ export class ServiceBusClient {
   }
 
   /**
-   * Creates a receiver for an Azure Service Bus queue.
+   * Creates a receiver for an Azure Service Bus queue in peekLock mode.
+   *
+   * In peekLock mode, the receiver has a lock on the message for the duration specified on the
+   * queue or subscription.
+   *
+   * If the message is not settled during this time it becomes available to other receivers.
```

Comment:
Consider adding the below 

> You can settle a message in one of 4 ways based on your needs by calling either of `complete()`, `abandon()`, `defer()` or `deadletter()` methods on the message.

Reply:
Love it, great idea!

Reply:
Hemingway'd a bit but otherwise intact. Pushed.